src/core/Form/MultiSelect/MultiSelect/MultiSelect.tsx

```diff
@@ -303,25 +315,36 @@ class BaseMultiSelect<T> extends Component<
     this.setState({ showPopover: visible });
   };
 
-  private handleBlur = (event: React.FocusEvent<HTMLInputElement>) => {
-    event.preventDefault();
+  private getOwnerDocument = () => {
     if (this.popoverListRef !== null && this.popoverListRef.current !== null) {
       const elem = this.popoverListRef.current;
       const ownerDocument = windowAvailable()
         ? elem
           ? elem.ownerDocument
           : document
         : null;
+      return ownerDocument;
+    }
+    return null;
+  };
+
+  private focusInInput = (ownerDocument: Document | null) =>
+    ownerDocument
+      ? ownerDocument.activeElement === this.filterInputRef.current
+      : false;
 
+  private handleBlur = (event: React.FocusEvent<HTMLInputElement>) => {
+    event.preventDefault();
+    if (this.popoverListRef !== null && this.popoverListRef.current !== null) {
```

Comment:
This seems to be unnecessary as there is a guard clause just below doing the exact same check and returning if either of the checked values is null.

Reply:
Good find. Removing this in next fix branch for MultiSelect.